garden_ai/app/notebook.py

```diff
@@ -363,7 +363,10 @@ def debug(
         )
 
         image = build_notebook_session_image(
-            docker_client, path, local_base_image_id, pull=False
+            docker_client,
+            path,
+            local_base_image_id,
+            # env_vars={"GARDEN_SKIP_TESTS": False},  # TODO: what's better UX for debugging?
```

Comment:
this was a change I made thinking it might be more convenient, but this could also make the debug image inconsistent with the actual published image